docs/docs/recipes/organizations/organization-invitations.mdx

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 5
+---
+
+# Invite members to your organization
+
+Organizations are a way to group users and resources together. You can invite users to your organization to collaborate on projects and resources.
+
+## Create organization roles
+
+Before inviting members to your organization, you need to create organization roles. Check out the [Configure organizations](/docs/recipes/organizations/configuration/) guide for detailed instructions.
+
+In this guide, let's create two typical organization roles: `admin` and `member`.
+
+The `admin` role has full access to all resources in the organization, while the `member` role has limited access. For example, each role can have a set of permissions as follows:
+
+- `admin` role:
+  - `read:data` - Read access to all organization data resources.
+  - `write:data` - Write access to all organization data resources.
+  - `delete:data` - Delete access to all organization data resources.
+  - `invite:member` - Invite members to the organization.
+  - `manage:member` - Manage members in the organization.
+  - `delete:member` - Remove members from the organization.
+- `member` role:
+  - `read:data` - Read access to all organization data resources.
+  - `write:data` - Write access to all organization data resources.
+  - `invite:member` - Invite members to the organization.
+
+## Configure your email connector
+
+Since invitations are sent via email, ensure your email connector is properly configured. To send invitations, you need to configure a newly introduced email template usage type - `OrganizationInvitation`.
+
+:::note
+Logto Cloud's built-in "Logto email service" does not support the `OrganizationInvitation` usage type at the moment. Instead, you need to configure your email connector (e.g. Sendgrid) and set up the `OrganizationInvitation` template.
+:::
+
+A sample email template for the `OrganizationInvitation` usage type is shown below:
+
+```json
+{
+  "subject": "Welcome to my organization",
+  "content": "<p>Join my organization by this <a href=\"{{link}}\" target=\"_blank\">link</a>.</p>",
+  "usageType": "OrganizationInvitation",
+  "type": "text/html"
+}
+```
+
+The `{{link}}` placeholder in the email content will be replaced with the actual invitation link when sending the email. In this guide, let's say it would be `https://your-app.com/invitation/accept/your-invitation-id`.
```

Comment:
`your-invitation-id` seems to be a placeholder of an variable, better using brackets to wrap it up?

Reply:
updated